### PR TITLE
Updating ansible steps for clarity

### PR DESCRIPTION
**Pull request by kickoke**
Fixed typos. 

However, the document needs some refinement (adding to my backlog). 
The document contains steps in form of paragraphs, instead of using ordered lists. 
It is also a bit generic at times, for example lines 49-51. 
"First, download the
[playbook](https://github.com/netdata/community/tree/main/netdata-agent-deployment/ansible-quickstart), move it to the
current directory, and remove the rest of the cloned repository, as it's not required for using the Ansible playbook."

What does 'current' mean in this context? Which file should be edited and which ones deleted? This will be explained later in the text, but it's not obvious from the beginning.

### CODE REVIEW

**Comment by DShreve2** [comment]
> Fixed typos.
> 
> However, the document needs some refinement (adding to my backlog). The document contains steps in form of paragraphs, instead of using ordered lists. It is also a bit generic at times, for example lines 49-51. "First, download the [playbook](https://github.com/netdata/community/tree/main/netdata-agent-deployment/ansible-quickstart), move it to the current directory, and remove the rest of the cloned repository, as it's not required for using the Ansible playbook."
> 
> What does 'current' mean in this context? Which file should be edited and which ones deleted? This will be explained later in the text, but it's not obvious from the beginning.

@kickoke: We'll approach some of these refinement issues together. I'm going to approve this PR, but before we squash and merge (we'll need at least one more review), let's make the title a little more descriptive so when the commit is collected later, we know what the change is. Even just "Updating ansible steps for clarity" will suffice.

**Review by DShreve2 — APPROVED** [review]

**Review by ilyam8 — APPROVED** [review]

**Pull request merged**
